recipe/meta.yaml

```diff
@@ -142,6 +146,10 @@ test:
         - test -f $PREFIX/lib/libopencv_freetype${SHLIB_EXT}     # [unix]
         - python -c "import cv2; assert 'Unknown' not in cv2.videoio_registry.getBackendName(cv2.CAP_V4L)"  # [linux]
         - python -c "import cv2, re; assert re.search('Lapack:\s+YES', cv2.getBuildInformation())"
+        - mkdir -p cmake_build_test && pushd cmake_build_test
+        - cmake -G "Ninja" -D "OpenCV_DIR={{ PREFIX }}/cmake" ..
```

Comment:
why do we need this? don't we want cmake to be able to find this automatically? without the need for any kinds of defines from the user?

Reply:
Thanks for fixes!
Yes, I'll remove it.